### PR TITLE
dcos: add simulate JVM application faults.md

### PR DESCRIPTION
**Pull request by Liuxiaozhen12**
Signed-off-by: Liuxiaozhen12 <82579298+Liuxiaozhen12@users.noreply.github.com>
This PR is translated from https://github.com/chaos-mesh/website-zh/pull/32.

### CODE REVIEW

File: docs/simulate-jvm-application-chaos.md [hunk 1]
```diff
@@ -1,6 +1,187 @@
...
+
+For the meaning of the value of action, refer to:
+
+| Name   | Meaning                                                         |
+| ------ | --------------------------------------------------------------- |
+| delay  | Specifies method call delay                                     |
+| return | Writes groovy and Java implement scenarios                      |
+| script | Writes groovy and Java implement scenarios                      |
+| cfl    | Java CPU usage overload                                         |
+| oom    | Out of memory, supporting oom of piles, stacks, metaspace areas |
```
Comment (STRRL) [inline docs/simulate-jvm-application-chaos.md:178]:
```suggestion
| oom    | Out of memory, supporting oom of heap, stack, metaspace |
```

JVM 里的堆内存特指 “heap memory”(算是专有名词？), 后面的两个也是, 应该直接放到那里就可以了吧。

File: docs/simulate-jvm-application-chaos.md [hunk 2]
```diff
@@ -1,6 +1,187 @@
...
+| ccf    | JVM code cache fill                                             |
+| tce    | Throw custom exceptions                                         |
+| cpf    | Connect pool full                                               |
```
Comment (STRRL) [inline docs/simulate-jvm-application-chaos.md:181]:
```suggestion
| cpf    | Connection pool full                                               |
```
连接池 “connection pool”, 应该也是个专有名词

File: docs/simulate-jvm-application-chaos.md [hunk 3]
```diff
@@ -1,6 +1,187 @@
...
+| tde    | The first exception in a throw method                           |
```
Comment (STRRL) [inline docs/simulate-jvm-application-chaos.md:182]:
```suggestion
| tde    | Throw the first exception of method declare                          |
```

File: docs/simulate-jvm-application-chaos.md [hunk 4]
```diff
@@ -1,6 +1,187 @@
...
+| tpf    | Thread pool full                                                |
+
+For the details of action, refer to [chaos blade document](https://chaosblade-io.gitbook.io/chaosblade-help-zh-cn/blade-create-jvm).
+
+For the parameters passed to `chaosblade-exec-jvm`, Chaos Mesh will merge all fields in `flags` and `matchers` as a request body and then send it to `chaosblade-exec-jvm`. For details, refer to [chaosblade-exec-jvm/Agreements](https://github.com/chaosblade-io/chaosblade-dev-doc/blob/a7074ab656de469f7dfaa19227723d0967c590ae/zh-cn/chaosblade-exec-jvm/%E5%8D%8F%E8%AE%AE%E7%AF%87.md).
```
Comment (STRRL) [inline docs/simulate-jvm-application-chaos.md:187]:
```suggestion
For the parameters passed to `chaosblade-exec-jvm`, Chaos Mesh will merge all fields in `flags` and `matchers` as a request body and then send it to `chaosblade-exec-jvm`. For details, refer to [chaosblade-exec-jvm/Protocol](https://github.com/chaosblade-io/chaosblade-dev-doc/blob/a7074ab656de469f7dfaa19227723d0967c590ae/zh-cn/chaosblade-exec-jvm/%E5%8D%8F%E8%AE%AE%E7%AF%87.md).
```

**Review by STRRL — CHANGES_REQUESTED** [review]
Thanks @Liuxiaozhen12! PTAL! :D

File: docs/simulate-jvm-application-chaos.md [hunk 5]
```diff
@@ -1,6 +1,187 @@
...
+| target    | string            | Indicates the parameter passed to `chaosblade-exec-jvm`, representing JVMChaos targets, supporting servlet, psql, jvm, jedis, http, dubbo, rocketmq, tars, mysql, ruid, redisson, rabbitmq, monodb. | None          | Yes      | jvm     |
+| flags     | map[string]string | Indicates parameters passed to `chaosblade-exec-jvm` and represents the flags of action.                                                                                                            | None          | No       |         |
+| matchers  | map[string]string | Indicates parameters passed to `chaosblade-execu-jvm` and represents the matching of injection points.                                                                                              | None          | No       |         |
+
+For the meaning of the value of action, refer to:
+
+| Name   | Meaning                                                         |
+| ------ | --------------------------------------------------------------- |
+| delay  | Specifies method call delay                                     |
+| return | Writes groovy and Java implement scenarios                      |
```
[the remark this answers is not in the document]
Comment (Liuxiaozhen12) [reply docs/simulate-jvm-application-chaos.md]: This problem has been solved here: https://github.com/chaos-mesh/website/pull/101/commits/3ca3824612e72e44e80b865ad3de60ca3e5cf73b. Shall I fix it in the Chinese document?

**Comment by Liuxiaozhen12** [comment]
@STRRL Thanks for your detailed review! PTAL again~

File: docs/simulate-jvm-application-chaos.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (STRRL) [reply docs/simulate-jvm-application-chaos.md]: I have fixed that in https://github.com/chaos-mesh/website-zh/pull/58. :D

File: docs/simulate-jvm-application-chaos.md [hunk 6]
```diff
@@ -1,6 +1,187 @@
...
+Prepare a template for modifications to be made by JVMChaos:
+
+```shell
+kubectl apply -f sidecar-template.yaml
+kubectl apply -f sidecar.yaml
+```
+
+### 2. Create tested applications
+
+[jvm-chaos-demo](https://github.com/chaos-mesh/jvm-chaos-demo) is a simple Spring Boot application and here serves as a target application. A tested application is defined in `example/jvm/app.yaml` as follows:
```
Comment (STRRL) [inline docs/simulate-jvm-application-chaos.md:55]:
```suggestion
### 2. Create target applications

[jvm-chaos-demo](https://github.com/chaos-mesh/jvm-chaos-demo) is a simple Spring Boot application and here serves as a target application. A target application is defined in `example/jvm/app.yaml` as follows:
```

**Review by STRRL — COMMENTED** [review]
rest LGTM!

**Review by STRRL — APPROVED** [review]

**Pull request merged**